content/docs/studio/index.md

```diff
@@ -0,0 +1,26 @@
+# DVC Studio
+
+[`DVC Studio`](https://viewer.iterative.ai/) is a comprehensive and interactive
+collboration tool for your Machine Learning projects.
+
+Using the power of leading open-source tools DVC, CML and Git, it enables you to
+seamlessly manage data and models, run and track experiments, and visualize and
+share results.
+
+Project website: https://viewer.iterative.ai
```

Comment:
Probably no need for this since it's linked in the p above and Idk if we want to spell out the `viewer` subdomain here as I guess it's likely to change.

```suggestion
```

Reply:
Leaving this extra link, but correcting it to https://studio.iterative.ai

Reply:
So should it be studio.iterative.ai everywhere? (In the md links)

Reply:
yes.. let me know if you find viewer.iterative.ai anywhere

Reply:
Nvmd I see you already changed the old ULRs!